quinn-proto/src/endpoint.rs

```diff
@@ -861,6 +862,15 @@ pub struct TransportConfig {
     pub loss_reduction_factor: u16,
     /// Number of consecutive PTOs after which network is considered to be experiencing persistent congestion.
     pub persistent_congestion_threshold: u32,
+    /// Number of keep-alive packets to send before a connection times out.
```

Comment:
My intuition is that this is usually configured as a period rather than a frequency. Why does it work this way? Can we make it easier to use?